### DISCUSSION TITLE
Training a new organism's genome

### DISCUSSION THREAD

**Discussion by DiyaPrabhuram** [Gaius-Augustus/Augustus#336] [Q&A] [answered]
Hi!

I have the genome of soybean (Glycine max). I read through the command-line docs and it said augustus requires two mandatory parameters to run - species and queryfile. I have the query file in fasta format but soybean is not part of the augustus species repertoire. Any suggestions on what to do in a situation like this? I believe I'm supposed to re-train but I don't know how to go about doing that.

Thank you in advance!

Cheers,
Diya

**Comment by KatharinaHoff** [comment, marked as answer]
AUGUSTUS has been trained for soybean, just not by our group, and I don't
think we were contacted to share the parameter set. You could however get
in touch with the authors of these papers and request parameters. A simple
google search reveals at least two papers from last year.

Otherwise, you can try BRAKER either with RNA-Seq or with protein data for
training AUGUSTUS in a fully automated way. Be sure to softmask the genome
for repeats, beforehand.

**Reply by DiyaPrabhuram** [reply]
That's super helpful. Thank you!
